.github/workflows/links_checker.yml

```diff
@@ -77,4 +77,4 @@ jobs:
         if: ${{ steps.lychee.outputs.exit_code == 0 }}
         uses: peter-evans/close-issue@v3
         with:
-          issue-number: ${{ steps.last_issue.outputs.issue_number }}
+          issue-number: ${{ steps.last_issue.outputs.issue-number }}
```

Comment:
Thanks for the fix, @after-ephemera.

There are some other instances of `issue_number`, probably those should be replaced as well, right? Also, there is a `has_found` that might also rather be a `has-found`?